### ISSUE TITLE
Pattern submission: social gallery

### ISSUE THREAD

**Issue by karmatosed** [WordPress/pattern-directory#171]
Often people collect images that they create or find through social media. It might even be a #hashtag. This pattern has a heading you can use for that tag, or just words, a way to display those curated images and a social icon section to further follow.

It includes spacers to allow that 'just fitting' customization, full with and some simple styling.

---

## Pattern Title
A gallery that includes social icons

## Pattern Categories
Gallery, Social icons, Heading

## Screenshots
![image](https://user-images.githubusercontent.com/253067/121699125-92ca5100-cac6-11eb-9a87-7f223eedd9ae.png)

![image](https://user-images.githubusercontent.com/253067/121699211-a5dd2100-cac6-11eb-84db-d7f97bbd86e5.png)

![image](https://user-images.githubusercontent.com/253067/121699446-d6bd5600-cac6-11eb-8de4-ac1134ec6f51.png)

## Image Credits
https://www.rawpixel.com/image/556444/vintage-victorian-house-illustration
https://www.rawpixel.com/image/573354/antique-binoculars-drawing
https://www.rawpixel.com/image/572893/ship-with-sails
https://www.rawpixel.com/image/556653/vintage-guinea-pig-illustration
https://www.rawpixel.com/image/573505/suitcase-old-style
https://www.rawpixel.com/image/556532/vintage-umbrella-illustration

## Block Markup
```
<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"50px","right":"10px","bottom":"0px","left":"10px"}}},"backgroundColor":"white"} -->
<div class="wp-block-group alignfull has-white-background-color has-background" style="padding-top:50px;padding-right:10px;padding-bottom:0px;padding-left:10px"><div class="wp-block-group__inner-container"><!-- wp:spacer {"height":2} -->
<div style="height:2px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->

<!-- wp:heading {"textAlign":"center","align":"full","fontSize":"extra-large"} -->
<h2 class="alignfull has-text-align-center has-extra-large-font-size"><strong>#ENGRAVINGS</strong></h2>
<!-- /wp:heading -->

<!-- wp:spacer {"height":13} -->
<div style="height:13px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->

<!-- wp:gallery {"ids":[44,52,47,54,69,56],"linkTo":"none","align":"wide"} -->
<figure class="wp-block-gallery alignwide columns-3 is-cropped"><ul class="blocks-gallery-grid"><li class="blocks-gallery-item"><figure><img src="http://patterns.local/wp-content/uploads/2021/06/image-from-rawpixel-id-556444-jpeg-1024x1024.jpg" alt="" data-id="44" data-full-url="http://patterns.local/wp-content/uploads/2021/06/image-from-rawpixel-id-556444-jpeg.jpg" data-link="http://patterns.local/38-2/image-from-rawpixel-id-556444-jpeg/" class="wp-image-44"/></figure></li><li class="blocks-gallery-item"><figure><img src="http://patterns.local/wp-content/uploads/2021/06/image-from-rawpixel-id-572893-jpeg-1024x1024.jpg" alt="" data-id="52" data-full-url="http://patterns.local/wp-content/uploads/2021/06/image-from-rawpixel-id-572893-jpeg.jpg" data-link="http://patterns.local/38-2/image-from-rawpixel-id-572893-jpeg/" class="wp-image-52"/></figure></li><li class="blocks-gallery-item"><figure><img src="http://patterns.local/wp-content/uploads/2021/06/image-from-rawpixel-id-556532-jpeg-1024x1024.jpg" alt="" data-id="47" data-full-url="http://patterns.local/wp-content/uploads/2021/06/image-from-rawpixel-id-556532-jpeg.jpg" data-link="http://patterns.local/38-2/image-from-rawpixel-id-556532-jpeg/" class="wp-image-47"/></figure></li><li class="blocks-gallery-item"><figure><img src="http://patterns.local/wp-content/uploads/2021/06/image-from-rawpixel-id-573354-jpeg-1024x1024.jpg" alt="" data-id="54" data-full-url="http://patterns.local/wp-content/uploads/2021/06/image-from-rawpixel-id-573354-jpeg.jpg" data-link="http://patterns.local/38-2/image-from-rawpixel-id-573354-jpeg/" class="wp-image-54"/></figure></li><li class="blocks-gallery-item"><figure><img src="http://patterns.local/wp-content/uploads/2021/06/Group-1-1024x795.jpg" alt="" data-id="69" data-full-url="http://patterns.local/wp-content/uploads/2021/06/Group-1-scaled.jpg" data-link="http://patterns.local/38-2/group-1/" class="wp-image-69"/></figure></li><li class="blocks-gallery-item"><figure><img src="http://patterns.local/wp-content/uploads/2021/06/image-from-rawpixel-id-573505-jpeg-1024x1024.jpg" alt="" data-id="56" data-full-url="http://patterns.local/wp-content/uploads/2021/06/image-from-rawpixel-id-573505-jpeg.jpg" data-link="http://patterns.local/38-2/image-from-rawpixel-id-573505-jpeg/" class="wp-image-56"/></figure></li></ul></figure>
<!-- /wp:gallery -->

<!-- wp:social-links {"size":"has-large-icon-size","align":"center","className":"is-style-twentytwentyone-social-icons-color"} -->
<ul class="wp-block-social-links aligncenter has-large-icon-size is-style-twentytwentyone-social-icons-color"><!-- wp:social-link {"url":"http://instagram.com","service":"instagram"} /-->

<!-- wp:social-link {"url":"http://twitter.com","service":"twitter"} /-->

<!-- wp:social-link {"url":"http://wordpress.com","service":"wordpress"} /--></ul>
<!-- /wp:social-links -->

<!-- wp:spacer {"height":45} -->
<div style="height:45px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer --></div></div>
<!-- /wp:group -->

<!-- wp:paragraph -->
<p></p>
<!-- /wp:paragraph -->
```

**Comment by beafialho** [comment]
Thanks @karmatosed! I think this one is ready to be added to the directory :ship:

**Comment by beafialho** [comment]
I've added the pattern here: https://wordpress.org/patterns/pattern/social-gallery/

**Comment by beafialho** [comment]
Closed #171.
